src/UI/GameOver/Initials/Initials.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import styles from './Initials.css';
+
+export default ({ onInitialsChanged }) => {
+  const [initials, setInitials] = useState('');
+
+  const handleInitialsChange = event => {
+    setInitials(event.target.value);
+  };
+
+  const handleInitialsKeyPress = event => {
+    if (event.key === 'Enter') {
+      onInitialsChanged(initials.toUpperCase());
+    }
+  };
+
+  return (
+    <div className='d-flex flex-row mb-5 align-items-center'>
+      <div className='text-white' style={styles.label}>
+        Enter your initials:
```

Comment:
thoughts on `Enter first name: ` and maxLength it to like 7 or 8?

Reply:
Sure! Or we can do "Enter name" and limit it to 7 or 8 too. Which do you prefer?

Reply:
ah, probably "Enter Name" and trim before any whitespace, and limit to 7 or 8ish.  What do you think?

Reply:
Updated to use name input